images/kubekins-test/Dockerfile-1.11

```diff
@@ -43,6 +43,40 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get install -y \
 	--no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
+
+
+# Install Docker deps, some of these are already installed in the image but
+# that's fine since they won't re-install and we can reuse the code below
+# for another image someday.
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common \
+    lsb-release
+
+# Add the Docker apt-repository
+RUN curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg \
+    | apt-key add - && \
+    add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+    $(lsb_release -cs) stable"
+
+# Install Docker
+# TODO(bentheelder): the `sed` is a bit of a hack, look into alternatives.
+# Why this exists: `docker service start` on debian runs a `cgroupfs_mount` method,
+# We're already inside docker though so we can be sure these are already mounted.
+# Trying to remount these makes for a very noisy error block in the beginning of
+# the pod logs, so we just comment out the call to it... :shrug:
+# TODO(benthelder): update docker version. This is pinned because of
+# https://github.com/kubernetes/test-infra/issues/6187
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends docker-ce=17.09.1~ce-0~debian && \
+    sed -i 's/cgroupfs_mount$/#cgroupfs_mount\n/' /etc/init.d/docker
+
+
```

Comment:
Do we want to cleanup like we do in the cross image?

```
apt-get clean && rm -rf /var/lib/apt/lists/*
```

Reply:
maybe? we should probably do that in another PR though since it takes time to build + push the image and we'd also want to change this at the source (images/bootstrap/Dockerfile) which affects a *lot* more jobs.

Reply:
yep sounds good

Reply:
opening an issue to track